data-processing/Dockerfile

```diff
@@ -63,7 +63,7 @@ WORKDIR /data-processing
 COPY . .
 
 # Set default Python to Python 3
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
```

Comment:
@kyleclo can you try re-running the Docker build with this fix? It appears that Ubuntu had Python 3.6 installed by default, and the system Python was getting set to Python 3.6 instead of 3.7. This should fix that.